db_sql.go

```diff
@@ -58,6 +58,11 @@ func (x *sqlUserStoreDB) Authenticate(identity, password string, authTypeCheck A
 		return ErrIdentityAuthNotFound
 	}
 
+	var accountLocked bool
+	if err := x.db.QueryRow(`SELECT coalesce(accountlocked, false) FROM authuserpwd WHERE userid = $1`, userId).Scan(&accountLocked); err != nil || accountLocked {
```

Comment:
You're doing the exact same query as below, just with different fields returned. Just combine them into one query.